rules/legacy_xcodeproj.bzl

```diff
@@ -730,15 +732,15 @@ _BUILD_WITH_BAZEL_SCRIPT = """
 set -euxo pipefail
 cd $BAZEL_WORKSPACE_ROOT
 
-export BAZEL_DIAGNOSTICS_DIR="$BUILD_DIR/../../bazel-xcode-diagnostics/"
+export BAZEL_DIAGNOSTICS_DIR="{}"
```

Comment:
Can we use the key name here as `{BAZEL_DIAGNOSTICS_DIR}` to match the named param specified to the [format](https://bazel.build/rules/lib/core/string#format) method below?